### PR TITLE
Update avalon8-pmu-test.py

### PR DESCRIPTION
**Pull request by XzxEmbedded**

### CODE REVIEW

**Comment by XzxEmbedded** [comment]
please review the lastest commit, thanks.

File: scripts/avalon8/avalon8-pmu-test.py [hunk 1]
```diff
@@ -110,12 +115,15 @@ def detect_version():
...
-    if res[25:29] == PMU821_VER:
+    if res[14:16] == PMU821_VER:
         PMU_TYPE = PMU821_TYPE
-        PMU_VER = PMU821_VER
+        PMU_VER = res[14:29]
         PMU_ADC = PMU821_ADC
         PMU_LED = PMU821_LED
         PMU_PG  = PMU821_PG
+    else:
+        print("Invalid PMU version")
```
Comment (qinfengling) [inline scripts/avalon8/avalon8-pmu-test.py]: @XzxEmbedded Attach the Invalid version string. thanks.

File: scripts/avalon8/avalon8-pmu-test.py [hunk 2]
```diff
@@ -13,20 +13,25 @@
...
-PMU821_VER = ( '0dd0' )
+PMU821_VER = ( '8C' )
 
 PMU821_PG  = { 'pg_good': '0001', 'pg_bad': '0002' }
 
 PMU821_LED = { 'led_close': '0000', 'led_green': '0001', 'led_red': '0002' }
 
-PMU821_ADC = { 'ntc_l': 800, 'ntc_h': 1000, 'v12_l':931, 'v12_h': 1024, 'vcore_l': 927, 'vcore_h': 1024}
+#v12:vin, vcore:vout
+PMU821_ADC = { 'ntc_l': 524, 'ntc_h': 9615, 'v12_l':0, 'v12_h': 4095, 'vcore_l': 0, 'vcore_h': 4095}
```
Comment (qinfengling) [inline scripts/avalon8/avalon8-pmu-test.py]: @XzxEmbedded Add more comments about the ADC values. thanks.

**Comment by XzxEmbedded** [comment]
Please review pull request.

**Comment by Johnson-Fan** [comment]
Thinks

**Pull request closed without merging**